lib/kitto/router.ex

```diff
@@ -106,8 +106,12 @@ defmodule Kitto.Router do
 
   defp default_dashboard, do: Application.get_env(:kitto, :default_dashboard, "sample")
 
-  defp authenticate(conn) do
-    Application.get_env(:kitto, :auth_token) == conn
+  defp auth_token, do: Application.get_env(:kitto, :auth_token)
```

Comment:
What I typically do when I want to set something to the app env from system env is add the following to my `config/config.exs`:

```
config :kitto, auth_token: System.get_env("AUTH_TOKEN")
```

Whenever possible, I'm of the opinion that devs shouldn't use `System.get_env` in their application code.